OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -58,8 +67,13 @@
     @InjectView(R.id.prev_unit_title)
     private TextView mPreviousUnitLbl;
 
+    @com.google.inject.Inject
+    private CourseAPI courseApi;
```

Comment:
Please move to the import section in class

Reply:
done.